src/Behat/Behat/Context/Argument/SuiteScopedResolverFactory.php

```diff
@@ -18,6 +18,8 @@
  * @see ContextEnvironmentHandler
  *
  * @author Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * @deprecated since 3.4. Use `ArgumentResolverFactory` instead
```

Comment:
You should add a `trigger_error` with `E_DEPRECATED` on the file, as @stof suggested in https://github.com/Behat/Behat/pull/1018#discussion_r113420353

Reply:
ỳou cannot, as the implementation still implements it for BC